chain/index/README.MD

```diff
@@ -97,10 +97,12 @@ type IndexValidation struct {
 	TipSetKey TipSetKey
 	// Height is the epoch height at which the validation is performed.
 	Height uint64
-	// IndexedMessagesCount indicates the number of indexed messages for the canonical tipset at this epoch.
+	// IndexedMessagesCount is the number of indexed messages for the canonical tipset at this epoch.
 	IndexedMessagesCount uint64
-	// IndexedEventsCount signifies the number of indexed events for the canonical tipset at this epoch.
```

Comment:
@akaladarshi This file has changed. Please can you rebase this PR and then make the changes again ?

Reply:
Also, please can you update the docs for the  `IndexValidation` struct in `node/api/v1api/api_full.go`  ? You can see the docs I've added there for the `ChainValidateIndex` RPC API.

Reply:
Done.